icinga2x/Vagrantfile

```diff
@@ -65,6 +85,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
       # provider: virtualbox
       node_config.vm.provider :virtualbox do |vb, override|
+        node_config.ssh.private_key_path = "~/.ssh/id_rsa"
```

Comment:
Does this change make sense being added to other providers too? It doesn't seem related to OpenStack only.